setup.py

```diff
@@ -17,7 +17,7 @@ def read(filename: Union[Path, str]) -> str:
 
 setup(
     name="amsterdam-schema-tools",
-    version="0.17.9",
+    version="0.17.10",
```

Comment:
Rebase, please :)

Reply:
Done


> What does to_snake_case do now? Because that's where the error came from in [Amsterdam/dso-api#219](https://github.com/Amsterdam/dso-api/pull/219).

raises a `ValueError` now as well.